parser/class_files.py

```diff
@@ -81,22 +82,35 @@ def parse_images(tag, target, dita_soup):
     dita_images_title = dita_soup.new_tag("title")
     dita_images_title.string = "Images"
     dita_images.append(dita_images_title)
-    # loop through the HTML images and change them to dita
-    images = tag.find_all("img")
-    for img in images:
-        img_link = img["src"]
-        image_filename = os.path.basename(img_link)
-
-        # check it's not blacklisted
-        if not image_filename.lower() in black_list:
-            dita_image = dita_soup.new_tag("image")
-            dita_image["href"] = replace_characters(img_link, " ", "%20")
-            dita_image["scale"] = 33
-            dita_image["align"] = "left"
-            dita_images.append(dita_image)
-
-    # Append the dita <images> to the dita <body>
-    target.append(dita_images)
+
+    #Find the colspan:6 table
+    td = tag.find("td", {"colspan": "6"})
+
+    #Find the parent table (div id="Table")
+    if td is not None:
+        parent_table = td.find_parent("div", id="Table")
+
+        #Find the parent element of the table
+        page_layer = parent_table.find_parent("div", id="PageLayer")
```

Comment:
sadly I don't think we can rely on the id. We just find the immediate parent `div`, please.

Reply:
For line 91 we do need to provide the `id`.

For line 94, I think we can just use `parent_table.parent`.  That matches the patterns I see in the data.